compiler/ras/Debug.cpp

```diff
@@ -4952,8 +4952,8 @@ void TR_Debug::setupDebugger(void *addr)
          char * Argv[20];
 
          yield();
-         sprintf(cfname, "_%ld_", getpid());
-         sprintf(pp,"%ld", ppid);
+         sprintf(cfname, "_%" OMR_PRId64 "_", (int64_t)getpid());
+         sprintf(pp, "%" OMR_PRId64, (int64_t)ppid);
```

Comment:
We have a macro `INT64_PRINTF_FORMAT` that's defined as `"%" OMR_PRId64`.  I would recommend you use that instead.

Reply:
A problem with `INT64_PRINTF_FORMAT` is that it wouldn't allow one to specify width or alignmemt (e.g. if we wanted leading zeros, we could use `"_%016" OMR_PRId64 "_"` which you can't do with `INT64_PRINTF_FORMAT`). I'd go so far as to recommend removing _all_ uses of `INT64_PRINTF_FORMAT` in favour of `OMR_PRId64`.

Reply:
Fair enough.  I'll withdraw that suggestion.